packages/styled-components/src/components/primitives/Modal/Modal.types.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from 'react';
+import { Shapes } from 'components/Components.types';
+
+export type Props = {
+  customCss?: string;
+  overlayCustomCss?: string;
+  onClick?: () => void;
+  shape?: Shapes;
+  isOpen?: boolean;
+  children?: ReactNode;
+  closeButton?: boolean;
```

Comment:
Could we rename this to `showCloseButton` ?

Reply:
oops!, will do it in another PR